server/bin/pbench-server-prep-shim-002.py

```diff
@@ -311,51 +158,24 @@ def main(cfg_name):
 
     logger = get_pbench_logger(_NAME_, config)
 
-    # We're going to need the Postgres DB to track dataset state, so setup
-    # DB access.
-    init_db(config, logger)
-
-    qdir, receive_dir = fetch_config_val(config, logger)
+    receive_dir = fetch_config_val(config, logger)
 
-    if qdir is None and receive_dir is None:
+    if receive_dir is None:
         return 2
 
-    qdir_md5 = qdirs_check("quarantine", Path(qdir, "md5-002"), logger)
-    duplicates = qdirs_check("duplicates", Path(qdir, "duplicates-002"), logger)
-
-    # The following directory holds tarballs that are quarantined because
-    # of operational errors on the server. They should be retried after
-    # the problem is fixed: basically, move them back into the reception
-    # area for 002 agents and wait.
-    errors = qdirs_check("errors", Path(qdir, "errors-002"), logger)
+    counts = process_tb(config, logger, receive_dir)
 
-    if qdir_md5 is None or duplicates is None or errors is None:
-        return 1
-
-    counts = process_tb(config, logger, receive_dir, qdir_md5, duplicates, errors)
+    # FIXME: I believe we can just log the status instead of creating report
 
-    result_string = (
-        f"{config.TS}: Processed {counts.ntotal} entries,"
-        f" {counts.ntbs} tarballs successful,"
-        f" {counts.nquarantined} quarantined tarballs,"
-        f" {counts.ndups} duplicately-named tarballs,"
-        f" {counts.nerrs} errors."
+    logger.info("{}", counts.nstatus)
```

Comment:
I think this log message is redundant with the calls made at line 138.

Reply:
yeah :D